lib/routes/cffex/announcement.ts

```diff
@@ -0,0 +1,80 @@
+import { DataItem, Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+import cache from '@/utils/cache';
+
+export const route: Route = {
+    path: '/announcement',
+    name: '中国金融期货交易所 - 交易所公告',
```

Comment:
```suggestion
    name: '交易所公告',
```
Do not include namespace's name in route name